src/scitokens/scitokens.py

```diff
@@ -142,14 +142,15 @@ def verify(self):
         raise NotImplementedError()
 
 
-    def serialize(self, include_key=False, issuer=None, lifetime=600):
+    def serialize(self, include_key=False, issuer=None, lifetime=600, kid=None):
```

Comment:
Seems awkward to have `kid` here but the key itself is specified in the constructor.

Can you move `kid` to the constructor (and spell it `key_id` instead?  We don't use abbreviations for `issuer` here...)?